glslang/Include/Types.h

```diff
@@ -1565,16 +1448,11 @@ class TPublicType {
     const TType* userDef;
     TSourceLoc loc;
     TArraySizes* typeParameters;
-#ifndef GLSLANG_WEB
     // SPIR-V type defined by spirv_type directive
     TSpirvType* spirvType;
-#endif
 
-#ifdef GLSLANG_WEB
-    bool isCoopmat() const { return false; }
-#else
+
```

Comment:
Were these newlines there in the original code?

Reply:
Solved